packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -167,7 +160,7 @@ class AndroidCameraCameraX extends CameraPlatform {
   Future<List<CameraDescription>> availableCameras() async {
     final List<CameraDescription> cameraDescriptions = <CameraDescription>[];
 
-    processCameraProvider ??= await ProcessCameraProvider.getInstance();
+    processCameraProvider = await proxy.getProcessCameraProvider();
```

Comment:
Does this no longer need to be?

```suggestion
    processCameraProvider ??= await proxy.getProcessCameraProvider();
```